src/styles.scss

```diff
@@ -4,7 +4,7 @@ $brown-color: #756867;
 $accent-color: #ff8d3f;
 $light-grey-color: #d5d6d2;
 
-$primary-font-family: "Domine", sans-serif;
```

Comment:
Good practice to still use generic font families, as a fallback

Reply:
Yes, of course, I accidentally deleted it. I'll fix it.